From 9b9cf37e1f13655a8b2fa65444a0352c31c2a7b9 Mon Sep 17 00:00:00 2001
From: Yu Jiahua <yujiahua1@huawei.com>
Date: Thu, 20 Jan 2022 12:22:40 +0800
Subject: [PATCH] Revert "sched: Introcude config option
 SCHED_OPTIMIZE_LOAD_TRACKING"
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 9b9cf37e1f13655a8b2fa65444a0352c31c2a7b9


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4QU5Z?from=project-issue
CVE: NA

--------------------------------

This patch revert ias feature from open-euler kernel.

This reverts commit 44483ea0b5aa78fb9d449966b158c1ed4c3a9b18.

Signed-off-by: Yu Jiahua <Yujiahua1@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 init/Kconfig | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index f362f6fcc1d0..2373130a71bc 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -2080,14 +2080,6 @@ endmenu		# General setup
 
 source "arch/Kconfig"
 
-config SCHED_OPTIMIZE_LOAD_TRACKING
-	bool "Optimize scheduler load tracking"
-	default n
-	help
-	  Optimize scheduler load tracking, when load balance is not important
-	  in system, we close some load tracking in tick and enqueue or dequeue
-	  task, in this way, we can save some unnecessary cpu overhead.
-
 config RT_MUTEXES
 	bool
 
-- 
2.26.2

